src/test/java/codeu/model/store/basic/MessageStoreTest.java

```diff
@@ -75,6 +80,7 @@ public void testAddMessage() {
     messageStore.addMessage(inputMessage);
     Message resultMessage = messageStore.getMessagesInConversation(inputConversationId).get(0);
 
+    Assert.assertEquals(messageStore.getSize(), 4); //check size after adding message
```

Comment:
I feel you can add the testGetSize function into this test. Assert that size is 3 before sending the msg, and then check it again here to make sure it changed to 4 now.

Reply:
I changed the test to fit that model. I agree, I couldn't actually figure out the best way to test the multiple checks of size.